TreeDB/caching/leaf_page_stable_resource_test.go

```
@@ -0,0 +1,410 @@
+	if descriptor.Frontier().Bytes == 0 {
+		t.Fatal("captured frontier is empty")
+	}
+	if _, err := cached.valueLogIdentityPins.BeginDelete(descriptor.Identity()); !errors.Is(err, rootpublication.ErrResourcePinned) {
+		t.Fatalf("delete pinned raw segment error=%v want ErrResourcePinned", err)
+	}
+	stats := resources.Stats(time.Now())
+	if len(stats) != 1 || stats[0].NamespaceSyncs != 1 {
+		t.Fatalf("stable resource stats=%+v want one creation namespace sync", stats)
+	}
+
+	segmentPath, _, ok := newCachingLeafPageLog(cached, &cached.leafLog).(interface {
```

golangci-lint (2.12.2)

[error] right hand must be only type assertion

(forcetypeassert)